setup.py

```diff
@@ -64,6 +64,42 @@ def run(self):
 package_data += find_package_data(
     os.path.join('mypyc', 'lib-rt'), ['*.c', '*.h'])
 
+USE_MYPYC = False
+# To compile with mypyc, a mypyc checkout must be present on the PYTHONPATH
+if len(sys.argv) > 1 and sys.argv[1] == '--use-mypyc':
+    sys.argv.pop(1)
+    USE_MYPYC = True
+if os.getenv('MYPYC_USE_MYPYC', None) == '1':
+    USE_MYPYC = True
+
+if USE_MYPYC:
+  MYPYC_BLACKLIST = ('__init__.py')
```

Comment:
Did you mean to use a tuple?
```suggestion
  MYPYC_BLACKLIST = ('__init__.py',)
```